3-Python/task_3/task_3.py

```diff
@@ -0,0 +1,37 @@
+# Task 3
+
+# Self-study positional arguments for Python scripts (sys.argv). 
+# Write a script that takes a list of words (or even phrases)a Script should ask a user to write 
+# something to stdin until user won't provide one of argument phrases.
+
+import sys
+
+dictionary = list()
+for i in range(1, len(sys.argv)):
+    buf = sys.argv[i]
+    dictionary.append(buf)
+    for j in range(i+1, len(sys.argv)):
+        buf = ' '.join([buf, sys.argv[j]])
+        dictionary.append(buf)    
+    
+while True:
+
+    secret = input('Tell me your secret (word/phrase): ')
+
+    if secret in dictionary:
+        print('Nice! Your secret matches.')
+        exit()
+    elif secret in dictionary:
+        print('Nice! Your secret matches.')
+        exit()

```

Comment:
Why do you repeat exact same condition?

Reply:
Looks like a result of copy-paste process, can't remember what was the point, tbh :D 

I removed this repeated condition